apps/profiles/serializers/user_ratings.py

```diff
@@ -0,0 +1,53 @@
+from rest_framework import serializers
+from rest_framework.generics import get_object_or_404
+from apps.profiles.models.user_rate import UserRating, User
+
+
+class UserRatingCreateSerializer(serializers.ModelSerializer):
+    value = serializers.IntegerField(min_value=0, max_value=5, default=5)
+
+    class Meta:
+        model = UserRating
+        fields = ['value', 'comment']
+
+    def create(self, validated_data):
+        user = get_object_or_404(User, id=self.context["view"].kwargs["user_id"])
```

Comment:
Querying the database inside create is not a good practice, if you need them so much, do it in a view.